wpeverest/everest-forms
# Fix - Hide empty fields in the email.

### All Submissions:


### Changes proposed in this Pull Request:
Previously, when the field was empty, it would display in the email. This pull request addresses this issue and resolves it.

### How to test the changes in this Pull Request:

1. Create a form.
2. Submit the form with an empty field.
3. Verify if the empty field is shown or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Hide empty fields in the email.
